src/commands/lambda/flare.ts

```diff
@@ -40,105 +49,170 @@ export class LambdaFlareCommand extends Command {
         )
       )
-      errorFound = true
     }
 
     // Validate case ID
     if (this.caseId === undefined) {
-      this.context.stderr.write(commonRenderer.renderError('No case ID specified. [-c,--case-id]'))
-      errorFound = true
+      errorMessages.push(commonRenderer.renderError('No case ID specified. [-c,--case-id]'))
     }
 
     // Validate email
     if (this.email === undefined) {
-      this.context.stderr.write(commonRenderer.renderError('No email specified. [-e,--email]'))
-      errorFound = true
+      errorMessages.push(commonRenderer.renderError('No email specified. [-e,--email]'))
     }
 
-    if (errorFound) {
+    // Exit if there are errors
+    if (errorMessages.length > 0) {
+      this.context.stderr.write(errorMessages.join('') + '\n')
+
       return 1
     }
+    assert(this.functionName !== undefined)
 
     // Get AWS credentials
     this.context.stdout.write('\n🔑 Getting AWS credentials...\n')
     try {
       this.credentials = await getAWSCredentials()
     } catch (err) {
-      this.context.stderr.write(commonRenderer.renderError(err))
+      this.context.stderr.write(commonRenderer.renderError(err.message))
 
       return 1
     }
     if (this.credentials === undefined) {
-      this.context.stdout.write(commonRenderer.renderNoAWSCredentialsFound())
+      this.context.stdout.write('\n' + commonRenderer.renderNoAWSCredentialsFound())
       try {
         await requestAWSCredentials()
       } catch (err) {
-        this.context.stderr.write(commonRenderer.renderError(err))
+        this.context.stderr.write(commonRenderer.renderError(err.message))
 
         return 1
       }
     }
 
-    // Get Lambda function configuration
-    this.context.stdout.write('\n🔍 Getting Lambda function configuration...\n')
+    // Get and print Lambda function configuration
+    this.context.stdout.write('\n🔍 Fetching Lambda function configuration...\n')
     const lambdaClientConfig: LambdaClientConfig = {
       region,
       credentials: this.credentials,
     }
     const lambdaClient = new LambdaClient(lambdaClientConfig)
     let config
     try {
-      config = await getLambdaFunctionConfig(lambdaClient, this.functionName ?? '')
+      config = await getLambdaFunctionConfig(lambdaClient, this.functionName)
     } catch (err) {
       this.context.stderr.write(
         commonRenderer.renderError(`Unable to get Lambda function configuration: ${err.message}`)
       )
 
       return 1
     }
+    const configStr = util.inspect(config, false, undefined, true)
+    this.context.stdout.write(`\n${configStr}\n`)
 
-    // Print config
-    const configStrColored = util.inspect(config, false, undefined, true)
-    this.context.stdout.write(`\n${configStrColored}\n`)
-    if (this.isDryRun) {
-      this.context.stdout.write('\n🚫 The configuration was not sent as it was executed in dry run mode.\n')
+    // Get CloudWatch logs
+    const logs: [string, OutputLogEvent[]][] = []
```

Comment:
This typing is confusing. Also, why not use a map?

Reply:
I felt that a map was overkill / slower since we are only getting 3 log streams. However, I agree that the typing would be a lot more readable. Do you think I should change it to a map?

Reply:
Why would a map be an overkill? And if we are only getting three log streams, how would it be slower?

I think whatever is more readable is the way to go.

Reply:
Just because of the overhead of hashing and other calculations. Since we're not deleting entries or retrieving by key, I thought it would be unnecessary. But yes, using a map would have an unnoticeable difference in speed and be more readable.